modules/local/bowtie2_removal_build.nf

```diff
@@ -10,14 +10,14 @@ process BOWTIE2_REMOVAL_BUILD {
     path fasta
 
     output:
-    path 'bt2_index_base*', emit: index
-    path "versions.yml"   , emit: versions
+    path "*.bt2"        , emit: index
+    path "versions.yml" , emit: versions
 
     script:
     def args = task.ext.args ?: ''
     """
     mkdir bowtie
-    bowtie2-build --threads $task.cpus $fasta "bt2_index_base"
+    bowtie2-build --threads $task.cpus $fasta ${fasta.simpleName}
```

Comment:
Just my curiosity: does simpleName apply to channels as well?

Reply:
You can use it on any file object: https://www.nextflow.io/docs/latest/script.html#getting-file-attributes

Reply:
I'm going to merge in now, but if you have any other issues please shout